__tests__/news-server.test.js

```diff
@@ -76,6 +76,7 @@ describe("GET /api/articles/:article_id", () => {
           created_at: expect.any(String),
           votes: expect.any(Number),
           article_img_url: expect.any(String),
+          comment_count: expect.any(Number),
```

Comment:
I would have preferred that you made a new test for this rather than change an old test.